ChangeLog

```diff
@@ -1,3 +1,8 @@
+2022-01-27  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
```

Comment:
That's not what we use the ChangeLog file on the root for.
If it would be more important it should go to the NEWS file, but for now only the new dialect should go in there "in general".